pages/api/frontend/v0.1/stripe/checkoutSession.ts

```diff
@@ -30,7 +37,7 @@ export default async function handler(
 
   switch (req.method) {
     case "POST":
-      const stripe = new Stripe(stripeConfig.secretKey, {
+      const stripe = new Stripe(stripeConfig.secretKey as string, {
```

Comment:
Instead of just casting the type and ignoring the compiler, basically causing a runtime error, implement valid configuration handling

```suggestion
      if (!stripeConfig.secretKey) {
        throw new Error("Stripe secret key is not configured");
      }
      const stripe = new Stripe(stripeConfig.secretKey, {
```